apis/metal3.io/v1alpha1/hostupdatepolicy_types.go

```diff
@@ -0,0 +1,70 @@
+/*
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// HostUpdatePolicy enumerates the allowed host update policies
+type UpdatePolicy string
+
+const (
+	HostUpdatePolicyOnPreparing    UpdatePolicy = "onPreparing"
+	HostUpdatePolicyOnReboot       UpdatePolicy = "onReboot"	
+)
+
+// NOTE(janders) based on https://github.com/metal3-io/metal3-docs/pull/411/
```

Comment:
Please remove before finishing the draft

Reply:
Done.